src/main/kotlin/org/jetbrains/intellij/IntelliJPlugin.kt

```diff
@@ -687,6 +687,7 @@ abstract class IntelliJPlugin : Plugin<Project> {
             verificationReportsDir.convention(
                 project.layout.buildDirectory.dir("reports/pluginVerifier").map { it.asFile.canonicalPath }
             )
+            verificationReportsFormats.convention(listOf("plain", "html"))
```

Comment:
Those were supposed to be enum values

Reply:
Declared constants in `org.jetbrains.intellij.IntelliJPluginConstants`.